sonaranalyzer-dotnet/its/expected/ManuallyAddedNoncompliantIssues/NetFramework35-{965FCDD0-69E7-4635-80AF-DD135A94F477}-S1118.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
If these issues are not related to the current task, would be possible to fix them to reduce the noise?

Valid also for the rest of the *.json files.

Reply:
there's a lot of manual effort to do this, with the risk of human error

instead of this, I would rather invest in using the framework we have for the sonar C# front-end, where we parse the `//Noncompliant (S1234)` comments

this would mean to move the _Manually*_ tests to the Java ITs (to use the library that does the parsing and results testing), but it would make all these tests maintainable for the future

Reply:
I actually fixed them to reduce the noise in the PR

but I would really appreciate to have something that skips this step for the future :)